test/e2e/k8s_util.go

```diff
@@ -667,6 +668,22 @@ func (data *TestData) UpdateConfigMap(configMap *v1.ConfigMap) error {
 	return err
 }
 
+func (data *TestData) CreateConfigMap(namespace, name string, configData map[string]string, binaryData map[string]byte, immutable bool) (*v1.ConfigMap, error) {
```

Comment:
just model this function after `UpdateConfigMap`. Note that these functions are not super useful, but we can have `CreateConfigMap` for symmetry since we already have `UpdateConfigMap`.